src/pages/publication/public_publication/PublicPublicationContext.tsx

```diff
@@ -0,0 +1,98 @@
+import OpenInNewIcon from '@material-ui/icons/OpenInNew';
+import { DegreePublicationContext } from '../../../types/publication_types/degreePublication.types';
+import { ReportPublicationContext } from '../../../types/publication_types/reportPublication.types';
+
+const StyledContainer = styled.div`
+  display: flex;
+`;
+
+const StyledOpenInNewIcon = styled(OpenInNewIcon)`
+  margin-left: 0.25rem;
+`;
+
+export const PublicPublicationContextJournal: FC<{ publicationContext: JournalPublicationContext }> = ({
+  publicationContext,
+}) => {
+  const { t } = useTranslation('publication');
+  const { onlineIssn, title, url } = publicationContext;
+
+  return title ? (
+    <LabelContentRow minimal multiple label={`${t('references.journal')}:`}>
+      <StyledContainer>
+        <NormalText>{title}</NormalText>
+        {url && (
+          <Link href={url} target="_blank" rel="noopener noreferrer">
+            <StyledOpenInNewIcon aria-label={url} />
+          </Link>
+        )}
+      </StyledContainer>
+      {onlineIssn && `${t('references.issn')} ${onlineIssn}`}
+    </LabelContentRow>
+  ) : null;
+};
+
+export const PublicPublicationContextDegree: FC<{ publicationContext: DegreePublicationContext }> = ({
+  publicationContext,
+}) => {
+  const { t } = useTranslation('publication');
+  const { publisher, seriesTitle, url } = publicationContext;
+
+  return (
+    <>
+      {publisher && (
+        <LabelContentRow minimal multiple label={`${t('common:publisher')}:`}>
+          <StyledContainer>
+            <NormalText>{publisher}</NormalText>
+            {url && (
+              <Link href={url} target="_blank" rel="noopener noreferrer">
+                <StyledOpenInNewIcon aria-label={url} />
+              </Link>
+            )}
+          </StyledContainer>
+        </LabelContentRow>
+      )}
+      {seriesTitle && (
+        <LabelContentRow minimal label={`${t('references.series')}:`}>
+          {seriesTitle}
+        </LabelContentRow>
+      )}
+    </>
+  );
+};
+
+export const PublicPublicationContextReport: FC<{ publicationContext: ReportPublicationContext }> = ({
+  publicationContext,
+}) => {
+  const { t } = useTranslation('publication');
+  const { onlineIssn, publisher, seriesTitle, url } = publicationContext;
+
+  return (
+    <>
+      {publisher && (
+        <LabelContentRow minimal multiple label={`${t('common:publisher')}:`}>
+          <StyledContainer>
+            <NormalText>{publisher}</NormalText>
+            {url && (
+              <Link href={url} target="_blank" rel="noopener noreferrer">
+                <StyledOpenInNewIcon aria-label={url} />
+              </Link>
+            )}
+          </StyledContainer>
+          {onlineIssn && `${t('references.issn')} ${onlineIssn}`}
+        </LabelContentRow>
+      )}
+      {seriesTitle && (
+        <LabelContentRow minimal label={`${t('references.series')}:`}>
+          {seriesTitle}
+        </LabelContentRow>
+      )}
+    </>
```

Comment:
synd å måtte duplisere så mye kode i `PublicPublicationContextDegree` og `PublicPublicationContextReport`, men ser at det er vanskelig å slå sammen på en fornuftig måte også :/

Reply:
Ja, også tenker jeg at det godt kan hende at disse blir forskjellig fra hverandre senere igjen, så da er det like så greit å holde de adskilt fra starten av

Reply:
vi kunne flyttet ut det som skal se likt ut da, feks kunne begge brukt en felles komponent for å vise series, type `<DisplaySeriesTitle />`, eller tror du at sånne ting også kanskje en gang skal vises forskjellig basert på type?

Reply:
frykter det blir vanskelig å passe på at vi oppdaterer ting likt for alle typer hvis alt er eget

Reply:
jo, det er et godt poeng. Flytter det ut i egne komponenter 👍